### PR TITLE
Geronimp diamond

### PR DESCRIPTION
**Pull request by geronimp**
Fire away

### CODE REVIEW

File: graftm/hmmer.py [hunk 1]
```diff
@@ -535,136 +491,226 @@ def extract_orfs(self, input_path, raw_orf_path, hmmsearch_out_path, orf_titles_
...
-                for title in [x.split(' ')[0] for x in open(table).readlines() if not x.startswith('#')]:
-                    if title not in seen:
-                        output.write(str(title) + '\n')
-                        seen.append(title)
-                    else:
-                        pass       
-        
+
+            reads= set(
+                       list(
```
Comment (wwood) [inline graftm/hmmer.py]: I don't think this is necessary, yeh? Doesn't set just take an iterable?

File: graftm/hmmer.py [hunk 2]
```diff
@@ -535,136 +491,226 @@ def extract_orfs(self, input_path, raw_orf_path, hmmsearch_out_path, orf_titles_
...
-        '''Protein search pipeline - The searching step for the protein 
-        pipeline, where hits are identified in the input reads through 
-        HMMsearches
+        Parameters
+        ----------
+        search_result : obj
+            SequenceSearchResult object with all paramaters defined. Used here
+            to create rows containing information on alignment direction and 
+            alignment span.
+        r : int
```
Comment (wwood) [inline graftm/hmmer.py]: use real words for variables. Saves time in the long run, and really isn't that hard to type out

**Comment by wwood** [comment]
I'm failing tests:
```
ImportError: No module named regex
ETraceback (most recent call last):
  File "/home/ben/git/graftM/test/../bin/graftM", line 27, in <module>
    import graftm.run
  File "/home/ben/git/graftM/bin/../graftm/run.py", line 10, in <module>
    from graftm.hmmer import Hmmer
  File "/home/ben/git/graftM/bin/../graftm/hmmer.py", line 8, in <module>
    import regex
```

**Comment by geronimp** [comment]
first module load regex and try again. All tests passed for me before I merged.


________________________________

File: graftm/hmmer.py (hunk 1 above)
Comment (geronimp) [inline graftm/hmmer.py]: Correct, I'll fix it up

**Pull request closed without merging**